src/templates/base/base.html

```diff
@@ -244,7 +244,7 @@
     {% endif %}
 
     {% if ebook_size_in_mb and ebook_size_in_mb > 0 %}
-    <option value="/static/pdfs/web_almanac_{{ year }}_{{ lang }}.pdf" data-event="ebook-click">
+    <option value="https://github.com/HTTPArchive/almanac.httparchive.org/raw/main/src/static/pdfs/web_almanac_{{ year }}_{{ lang }}.pdf" data-event="ebook-click">
```

Comment:
Have you considered uploading the PDFs to Cloud Storage and directing users to cdn.httparchive.org?

Reply:
Is there a benefit to that, versus GitHub? It's still off our app engine.

Though it would be nice to get them out of GitHub as they are big and do make your repo quick big too every time we regenerate them. But to do that we'd need GitHub action to have Cloud Storage Access to be able to upload to there on generation and we've never been comfortable enough to add that to the repo given how many contributors we have.

Reply:
Well that turned out to be easier than I thought - done!